### DISCUSSION TITLE
Record per hour

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#551] [Q&A]
Hi all!

I was wondering, if i can get recording of both my Wyze V3 cams.
Does anybody know how?

My YML file, looks like:

==

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554/udp # RTSP
            - 8888:8888 # HLS
            - 5000:5000 # WEB-UI
            - 9997:9997 # Expose port for API
        environment:
            - WYZE_EMAIL=***@****.**
            - WYZE_PASSWORD=*******************
            - NET_MODE=P2P
            - QUALITY=HD80
            - RTSP_READTIMEOUT=10s
            - RTSP_WRITEIMEOUT=60s
            - KEEP_BAD_FRAMES=True
            - DEBUG_FRAMES=True
            - MAX-NOREADY=50
            - RTSP_PROTOCOLS=udp
            - FRESH_DATA=true
            - RTSP_API=yes

**Comment by JA16122000** [comment]
Oops!

I think i'll have to look @ this? 
https://github.com/mrlt8/docker-wyze-bridge/wiki/Stream-Recording#recording-configuration

**Comment by JA16122000** [comment]
Back!

I've got this so far now, but there are no files comping in this map.
==

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554/udp # RTSP
            - 8888:8888 # HLS
            - 5000:5000 # WEB-UI
            - 9997:9997 # Expose port for API
               environment:
            - WYZE_EMAIL=***@***.**
            - WYZE_PASSWORD=************
            - NET_MODE=P2P
            - QUALITY=HD80
            - RTSP_READTIMEOUT=10s
            - RTSP_WRITEIMEOUT=60s
            - KEEP_BAD_FRAMES=True
            - DEBUG_FRAMES=True
            - MAX-NOREADY=50
            - RTSP_PROTOCOLS=udp
            - FRESH_DATA=true
            - RTSP_API=yes
            - TZ=Europe/Amsterdam
            - RECORD_ALL=True 
            - RECORD_PATH=/home/pi/Record/{cam_name}
            - RECORD_FILE_NAME={cam_name}_%Y%m%d_%H_%M_%S
            - RECORD_LENGTH=900
        volumes:
            - /local/path/on/host/:/home/pi/Record/

What am I missing?
==

**Reply by mrlt8** [reply]
Could you try something like this:

```yaml
...
    - RECORD_ALL=True
    - RECORD_LENGTH=900
volumes:
    - /home/pi/Record/:/record/
```

**Reply by JA16122000** [reply]
version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554/udp # RTSP
            - 8888:8888 # HLS
            - 5000:5000 # WEB-UI
            - 9997:9997 # Expose port for API
        environment:
            - WYZE_EMAIL=***@***.**
            - WYZE_PASSWORD=***************
            - NET_MODE=P2P
            - QUALITY=HD80
            - RTSP_READTIMEOUT=10s
            - RTSP_WRITEIMEOUT=60s
            - KEEP_BAD_FRAMES=True
            - DEBUG_FRAMES=True
            - MAX-NOREADY=50
            - RTSP_PROTOCOLS=udp
            - FRESH_DATA=true
            - RTSP_API=yes
            - TZ=Europe/Amsterdam
            - RECORD_ALL=True 
            - RECORD_PATH=/home/pi/Record/{cam_name}
            - RECORD_FILE_NAME={cam_name}_%Y%m%d_%H_%M_%S
            - RECORD_LENGTH=900
        volumes:
            - /home/pi/Record/:/record/


Doesn't work eather, when i look in the folder on my raspberry pi
===

**Comment by JA16122000** [comment]
Working with your help!
I did change: - RECORD_PATH=/home/pi/Record/{cam_name}
to - RECORD_PATH=/Record/{cam_name}
